src/main/kotlin/com/github/mitchellolsthoorn/testgenie/evosuite/Runner.kt

```diff
@@ -20,7 +20,7 @@ class Runner {
             val javaPath = "java"// TODO: Source from config
 
             val pluginsPath = System.getProperty("idea.plugins.path")
-            val evoSuitePath = "$pluginsPath/TestGenie/lib/evosuite.jar"
+            val evoSuitePath = "$pluginsPath/TestGenie/lib/evosuite-1.0.2.jar"
```

Comment:
I think you had a variable for this version. Wouldn't it make sense to use it here?

Reply:
I also agree with this ☝